JavaAssignments/src/amarja/SquareCubeprog.java

```diff
@@ -0,0 +1,21 @@
+package amarja;
+//WAP to find square and cube of a number.
+public class SquareCubeprog {
+	static int square;
```

Comment:
Why this extra variables?
PLease update code as per suggestions and push it again.

Reply:
Removed unused variables from the program.